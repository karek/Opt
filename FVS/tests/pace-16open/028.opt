8
1 58 39 7 25 14 31 18 
