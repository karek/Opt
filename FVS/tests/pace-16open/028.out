8
12 1 31 29 58 10 27 54 
